### PR TITLE
IBAnimatable Playground - Transition Animations

### PR DESCRIPTION
**Pull request by JakeLin**
We have created a new playground to demonstrate the IBAnimatable features, currently, we can demo Transitions and Interactions, other features (UI and Animations) are coming.

Transition Animations  

In Transitions playground, we can see
1. all supported transitions
2. how they look like in Push/Pop and Present/Dismiss transitions
3. how supported segues with/without dismiss interaction works

In Interactions playground, we can see custom container transitions.

@tbaranes please have a look when you have time.

### CODE REVIEW

File: IBAnimatableApp/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# IBAnimatable Playground
+**IBAnimatable Playground** is a playground to see the features IBAnimatable can support for prototyping and designing UI, animations, transitions and interactions. You can find your prefer effects and apply them in Interface Builder. 
+
+The `code` folder is not mandatary for the demo App. **The app was made in Interface Builder with `IBAnimatable` without a single line of code**.
```
Comment (tbaranes) [inline IBAnimatableApp/README.md]: Typo: mandatary -> mandatory

File: IBAnimatableApp/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# IBAnimatable Playground
+**IBAnimatable Playground** is a playground to see the features IBAnimatable can support for prototyping and designing UI, animations, transitions and interactions. You can find your prefer effects and apply them in Interface Builder. 
+
+The `code` folder is not mandatary for the demo App. **The app was made in Interface Builder with `IBAnimatable` without a single line of code**. 
+
+Because we want to demostrate more effects (e.g. animations, transitions) with one Scene within Interface Builder. In most of Apps, we usually apply one effect to one View or Scene, then we can configurate that effect in Interface Builder without any code.
```
Comment (tbaranes) [inline IBAnimatableApp/README.md]: Typo: demostrate -> demonstrate

**Comment by tbaranes** [comment]
Awesome. You really did a great work on this! That will bring the demo to a new level!
Just some typos feedbacks, otherwise, all good 👍

**Comment by JakeLin** [comment]
@tbaranes thanks for your feedback, I am going to work on some small improvements e.g. adding gesture to some transition animators, adding `from` to system transition animators, deleting some unused segues. Then will start writing `Transitions.md`.

**Pull request merged**